ex10_html-css-fundamentals-part2

ruminmaxim opened the pull request:
1.The form with different values and inputs made according to the following design
2. The table made according to the necessary layout
3. Adaptive page made according to the psd template

ruminmaxim committed:
1.The form with different values and inputs made according to the following design
2. The table made according to the necessary layout
3. Adaptive page made according to the psd template

rakotoz commented on src/ex10_html-css-fundamentals-part2/style-02.css:
```diff
@@ -0,0 +1,33 @@
+table {
```
Не используй тэги в css

rakotoz commented on src/ex10_html-css-fundamentals-part2/style-03.css:
```diff
@@ -0,0 +1,466 @@
+
+.phone-icon,
+.map-icon,
+.mail-icon {
```
Используй один класс, вместо нескольких одинаковых

ruminmaxim replied:
@rakotoz ниже для каждого класса отдельно прописывается background-image. Или нужно сделать контейнер для иконки, а свг вставлять в html'e?

ruminmaxim replied:
@rakotoz сделал через свг в html

rakotoz commented on src/ex10_html-css-fundamentals-part2/style-03.css:
```diff
@@ -0,0 +1,466 @@
+
+/*MOBILE*/
+
+@media screen and (min-width: 320px) and (max-width: 960px) {
```
Вынеси медиа в отдельный файл

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.html:
```diff
@@ -0,0 +1,111 @@
+                <div class="contacts">
+                    <div class="phone-icon">
+                    </div>
+                    <div class="contacts-separator"></div>
```
используй стили, вместо отдельного тэга

ruminmaxim committed:
1. Table styles are fixed

Changes on the page:
1. Mobile stylesheet is separated now
2. The contact block is remaked fully
3. Bags fixes
